common/src/app/common/features.cljc

```diff
@@ -49,7 +49,8 @@
     "components/v2"
     "styles/v2"
     "layout/grid"
-    "plugins/runtime"})
+    "plugins/runtime"
+    "editor/v2"})
```

Comment:
Maybe the feature can be called, `text-editor/v2` ? I mean, editor means too many things.

Reply:
Sure, I'll change it